docs/manual-deployment.md

```diff
@@ -82,6 +50,13 @@ cp templates/core/.env.sample templates/core/.env
 | `TRE_ID` | A globally unique identifier. `TRE_ID` can be found in the resource names of the Azure TRE instance; for example, a `TRE_ID` of `mytre-dev-3142` will result in a resource group name for Azure TRE instance of `rg-mytre-dev-3142`. This must be less than 12 characters. Allowed characters: Alphanumeric, underscores, and hyphens. |
 | `ADDRESS_SPACE` | The address space for the Azure TRE core virtual network. |
 | `MANAGEMENT_API_IMAGE_TAG` | The tag of the Management API image. Make it the same as `IMAGE_TAG` above.|
+| `RESOURCE_PROCESSOR_CLIENT_ID` | The client (app) ID of a service principal with "Owner" role to the subscription as created above. Used by Resource Processor Function to deploy workspaces and workspace services. |
+| `RESOURCE_PROCESSOR_CLIENT_SECRET` | The client secret (app password) of a service principal with "Onwer" role to the subscription as created above. Used by the depl09oyment processor function to deploy workspaces and workspace services. |
+| `SWAGGER_UI_CLIENT_ID` | Generated when following auth guide. Client ID for swagger client to make requests. |
+| `AAD_TENANT_ID` | Generated when following auth guide. Tenant id against which auth is performed. |
+| `API_CLIENT_ID` | Generated when following auth guide. Client id of the "TRE API". |
+| `API_CLIENT_SECRET` | Generated when following auth guide. Client secret of the "TRE API". |
+| `DEPLOY_GITEA` | Set to true (default) to enable Git shared service. |
```

Comment:
I've also added this in my PR - along with creating a default. Don't mind if merge mine first or not but conflict will need sorting.

Reply:
I'll sort the conflict later.